### ISSUE TITLE
[A11Y] [VoiceOver] - Unable to select dropdown items with voiceover

### ISSUE THREAD

**Issue by Ajaz4193** [OfficeDev/office-ui-fabric-react#11532]
Environment Information
https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Browser and OS versions: (Safari and IOS 13.3)
-** Device: Iphone XR

Describe the issue:
VoiceOver -
Focus is not set to the dropdown options on expanding and hence unable to navigate to dropdown list items using voice over

Actual behavior:
Issue is once the dropdown is activated the list UI is displayed but the focus still is at the background control and user has to explicitly touch the list UI to navigate to the list elements.
Note: Same issue is not reproducible when the screen is in landscape mode.

Expected behavior:
Focus should move to the first item in the list as soon as the list items UI is displayed by default.
![IMG_0003](https://user-images.githubusercontent.com/40019907/71298580-a36c9180-233d-11ea-9839-674b9a1186e7.PNG)

**Comment by ecraig12345** [comment]
Thanks for reporting this. Have you checked whether the issue occurs on other platforms, such as desktop Safari + VoiceOver, and (on Windows) Edge + Narrator?

**Comment by Goopies** [comment]
This seems to work correctly in Edge + Narrator, I am also experiencing this bug in iOS as well.

**Comment by xugao** [comment]
same issue also reported in #11247. closing this one as duplicate.
